### ISSUE TITLE
Adopt an ORT project membership agreement

### ISSUE THREAD

**Issue by tsteenbe** [oss-review-toolkit/ort-governance#32]
I propose we work with Linux Foundation to adopt a ORT project membership agreement that organization can sign to give ORT permission to have their logo displayed on ORT's website or other marketing materials. 

Think we can re-use the recently introduced for the SPDX project membership agreement see https://cdn.platform.linuxfoundation.org/agreements/spdx.pdf.  We would omit point 3 but keep the rest.

Once the agreement is in place we can setup enrollment via [LFX](https://lfx.linuxfoundation.org/) so any existing Linux Foundation member can easily sign it via an interface they are already familiar with.

**Comment by itrich** [comment]
> > I propose we work with Linux Foundation
> 
> I'd actually prefer something that works for us without involving LF. Can't we just extend our [ADOPTERS](https://github.com/oss-review-toolkit/ort/blob/main/ADOPTERS.md) table with a "Logo" column, and if people add their company logo there, it signals us that we're good to use it?

We had the same discussion and ended up maintaining a `yml`-file to track all supporters: https://github.com/SovereignCloudStack/website/blob/main/_data/supporter.yml

Using a `yaml` rather than a `markdown` file had the gread side effect, that we can easily process the content with any arbitrary tool (e.g. our static site generator).

**Comment by tsteenbe** [comment]
> I'd actually prefer something that works for us without involving LF. Can't we just extend our [ADOPTERS](https://github.com/oss-review-toolkit/ort/blob/main/ADOPTERS.md) table with a "Logo" column, and if people add their company logo there, it signals us that we're good to use it?

People who make pull requests often are not authorized within their organization to sign off on letting others us their logo so no a extension to ADOPTERS.md won't suffice. Need a legally binding document we can rely on. LF has all the infrastructure in place to make this very easy for ORT users and the ORT project.

**Comment by sschuberth** [comment]
> People who make pull requests often are not authorized within their organization to sign off on letting others us their logo

I don't think that matters. Those people could still contact the right people within their organization and then submit the logo on their behalf.

> Need a legally binding document we can rely on.

I believe according (legally binding) explanations in `ADOPTERS.md` should suffice. If SCS can do it without a legal process beyond committing to their `supporter.yml`, why can't we?
